openai.audio/client.bal

```diff
@@ -71,6 +72,7 @@ public isolated client class Client {
     # Translates audio into into English.
     #
     # + return - OK 
+    @display {label: "Create Translation"}
```

Comment:
Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?